application.yml

```diff
@@ -26,4 +26,7 @@ cf:
     password: admin
     skipSslValidation: false
     httpClient:
-      timeout: 6000ms
\ No newline at end of file
+      timeout: 6000ms
+authn:
```

Comment:
Should this authn settings be left like that? Is it leftover?

Reply:
they can be removed